.github/workflows/deploy-pages.yml

```diff
@@ -0,0 +1,24 @@
+name: Build and Deploy
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
:<

Reply:
yeah, the [post I was following](https://www.jameswright.xyz/post/deploy-hugo-academic-using-githubio/) says: "The proceeding steps rely on access to apt-get to install the required software to build the static site files, so Ubuntu is chosen."

I don't know what steps it talks about so I left it unchanged.